PRASFiles/test/runtests.jl

```diff
@@ -0,0 +1,30 @@
+using PRASCore
+using PRASFiles
+using Test
+
+@testset "PRASFiles" begin
+
+    @testset "Roundtrip .pras files to/from disk" begin
+
+        # TODO: Verify systems accurately depicted?
+        path = dirname(@__FILE__)
+
+        toy = toymodel()
+        savemodel(toy, path * "/toymodel2.pras")
+        toy2 = SystemModel(path * "/toymodel2.pras")
+        @test toy == toy2
+
+        rts = rts_gmlc()
+        savemodel(rts, path * "/rts2.pras")
+        rts2 = SystemModel(path * "/rts2.pras")
+        @test rts == rts2
+
+    end
+
+    @testset "Run RTS-GMLC" begin
+
+        assess(rts_gmlc(), SequentialMonteCarlo(samples=100), Shortfall())
```

Comment:
This was previously a test in `ResourceAdequacy` so I moved it here. I think it's a nice sanity check that the loaded system actually runs, but it could potentially be made more useful by testing that the RA outcomes are what was reported in the RTS-GMLC paper.

Reply:
That said, if memory serves there's actually a bug in this dataset where hydro is reported differently than the RTS repo, so results might not currently match... I haven't looked at that in a while though